docs/components/concepts/resource-authorizations.md

```diff
@@ -0,0 +1,31 @@
+---
+id: resource-authorizations
+title: "Resource authorizations"
+description: "In Camunda 8 SaaS, resource authorizations are supported to allow a finer grained approach to controlling access to your resources."
+---
+
+In Camunda 8 SaaS, resource authorizations allow you to control the level of access a user has to a specific resource in the system.
+
+### Component support
+
+The Camunda ecosystem contains several resource types and many actions that can be performed on them, currently the resource types and permissions we support are:
+
+#### Operate
+
+- Process definition:
+  - Read definition
+  - Delete definition
+  - Update process instance
+  - Delete process instance
+- Decision definition:
+  - Read
+  - Delete
+
+#### Tasklist
+
+- Process definition:
+  - Start process instance
```

Comment:
I like how this page is looking, but structurally (and possibly nitpicky), we should drop the headings/headers here unless we plan on adding more information (as in, more sentences) to these sections.

Reply:
Yea I wasn't too sure about the structure, I wanted to convey the relevancy of each permission with the context of where its applicable so headings felt like a nice way to do that but would you recommend just leaving the headings off and maybe emboldening the component names instead?

Reply:
I can open an issue in my backlog to give a little more love to the Self-Managed landing page, adding descriptions below the components and such, similar to my work for the [SaaS Components page](https://github.com/camunda/camunda-platform-docs/pull/2168). This should help remove the necessity of adding headers.

Reply:
Perfect, thank you @christinaausley, for now I will update this PR with my suggestion above to remove the headers for the moment